packages/web/src/components/gcds-header/gcds-header.css

```diff
@@ -1,28 +1,28 @@
 :host {
   display: block;
-  margin: 0 0 var(--gcds-spacing-500) !important;
+  margin: 0 0 var(--gcds-spacing-500) !important;/* --gcds-header-margin */
 
   .gcds-header__container {
     justify-content: space-between;
     width: 90%;
-    max-width: var(--gcds-container-400);
+    max-width: var(--gcds-header-container-max-width);
     margin: 0 auto;
   }
 
   .gcds-header__brand {
-    border-bottom: .1875rem solid var(--gcds-header-brand-border);
-    padding-bottom: var(--gcds-spacing-100);
-    margin-top: var(--gcds-spacing-300);
+    border-bottom: var(--gcds-header-brand-border-width) solid var(--gcds-header-brand-border-color);
```

Comment:
Can you please change `border-bottom` to `border-block-end` for better multi-language support?